File: patterns/c-patterns/pattern137.c
// Pattern 137: ...

// * * * *
//       *
//     *
//   *
// *

#include <stdio.h>

int main()
{
    int n = 5;
    int i, j;

    for (i = 1; i <= n; i++)
    {
        for (j = 1; j <= n - 1; j++)
        {
            if (i == 1 || i + j == n + 1)
            {
                printf("* ");
            }
            else
            {
                printf("  ");
            }
        }
        printf("\n");
    }
    return 0;
}

File: patterns/javascript-patterns/pattern137.js
// Pattern 137: ...

// * * * *
//       *
//     *
//   *
// *

let n = 5;

for (let i = 1; i <= n; i++) {
    let row = '';
    for (let j = 1; j <= n - 1; j++) {
        if (i === 1 || i + j === n + 1) {
            row += '* ';
        } else {
            row += '  ';
        }
    }
    console.log(row);
}
